bats_test/upload-file/single-bundled-files.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# Assertions
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+setup_file() {
+    run $DIR/create-file.sh
+    echo "##### Created 10 chunk test file" >>/home/node/bats.log
```

Comment:
Can we create a couple more files on this script to test with different chunk sizes?
Its a cheap way to greatly increase coverage